packages/vscode-extension/package.nls.json

```diff
@@ -362,40 +362,23 @@
     "teamstoolkit.webview.samplePageTitle": "Samples",
     "teamstoolkit.webview.surveyPageTitle": "Teams Toolkit Survey",
     "teamstoolkit.webview.accountHelp": "Account Help",
-    "teamstoolkit.taskDefinitions.command.prerequisites.description": "Validate and install prerequisites.\n See https://aka.ms/teamsfx-check-prerequisites-task for how to customize following args.",
-    "teamstoolkit.taskDefinitions.command.tab.description": "Set up tab.\n See https://aka.ms/teamsfx-debug-set-up-tab-task for how to customize following args.",
-    "teamstoolkit.taskDefinitions.command.bot.description": "Set up bot.\n See https://aka.ms/teamsfx-debug-set-up-bot-task for how to customize following args.",
-    "teamstoolkit.taskDefinitions.command.sso.description": "Set up SSO.\n See https://aka.ms/teamsfx-debug-set-up-sso-task for how to customize following args.",
-    "teamstoolkit.taskDefinitions.command.manifest.description": "Build and upload Teams manifest.\n See https://aka.ms/teamsfx-debug-prepare-manifest-task for how to customize following args.",
+    "teamstoolkit.taskDefinitions.command.prerequisites.description": "Validate prerequisites.\n See https://aka.ms/teamsfx-tasks/check-prerequisites for how to customize following args.",
```

Comment:
following -> the following
args -> arguments

Reply:
Thank you. Updated the wording.